### PR TITLE
Security rules hotfix

### PR DESCRIPTION
**Pull request by bgdeutsch**
### What does this PR do?
Quick fix to sec monitoring rules to allow docs to build again

### Motivation
Gitlab failures

### Preview
OOTB security rules and all other pulled sec monitoring content hasnt changed from live

https://docs-staging.datadoghq.com/brian.deutsch/buildfix/security_platform/default_rules/#all

### Additional Notes
---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: local/bin/py/build/actions/security_rules.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def security_rules(content, content_dir):
     logger.info("Starting security rules action...")
     global_aliases = []
     for file_name in chain.from_iterable(glob.glob(pattern, recursive=True) for pattern in content["globs"]):
+        print(file_name)
```
Comment (davidejones) [inline local/bin/py/build/actions/security_rules.py:74]: does this add a lot of noise?

**Review by davidejones — APPROVED** [review]

**Review by maycmlee — APPROVED** [review]

File: local/bin/py/build/actions/security_rules.py (hunk 1 above)
Comment (bgdeutsch) [reply local/bin/py/build/actions/security_rules.py]: removed, thank you!  @davidejones

**Comment by sarina-dd** [comment]
@bgdeutsch @davidejones Thanks for this fix! It looks like they merged the changes in the security monitoring repo and it removed the workload security option from the OOTB rules page with the change, too. Do ya'll mind taking a look?

**Comment by davidejones** [comment]
@sarina-dd @bgdeutsch I believe this pr should fix the issue if you can review/merge when ready. https://github.com/DataDog/documentation/pull/13067

**Pull request merged**